pkg/server/cache/entrycache/fullcache.go

```diff
@@ -88,8 +92,9 @@ type Agent struct {
 }
 
 type FullEntryCache struct {
-	aliases map[spiffeID][]aliasEntry
-	entries map[spiffeID][]*types.Entry
+	aliases     map[spiffeID][]aliasEntry
```

Comment:
I don't see any treatment of aliases with this code. We need to have a strategy to keep this up-to-date when node registration entries are mutated.